server/http_handler_test.go

```diff
@@ -320,6 +337,30 @@ func (ts *HTTPHandlerTestSuite) TestListTableRegions(c *C) {
 	c.Assert(err, IsNil)
 }
 
+func (ts *HTTPHandlerTestSuite) TestListTableRanges(c *C) {
```

Comment:
I think we can merge this test into TestRangesAPI?

Reply:
It's copied from TestListTableRegions which turns out to be a test for not existing table and a partitioned table which indeed is different test.